tests/workbookQueryTemplate.test.ts

```diff
@@ -40,7 +40,7 @@ describe("Single query template tests", () => {
         expect(path).toEqual(item1Path);
     });
 
-    test("ConnectedWorkbook XML exists as item2.xml", async () => {
+    test("ConnectedWorkbook XML exists as item1.xml", async () => {
```

Comment:
Why is this moved from items2 to items1? The new template saves connected-workbooks in items1?

Reply:
Yes, it was replaced after save